### PR TITLE
fix tests

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 4.2
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Review by chalasr — APPROVED** [review]

File: src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationDebugCommandTest.php [hunk 1]
```diff
@@ -68,7 +68,7 @@ public function testDebugDefaultDirectory()
     /**
      * @group legacy
      * @expectedDeprecation Storing translations in the "%ssf_translation%s/Resources/translations" directory is deprecated since Symfony 4.2, use the "%ssf_translation%s/translations" directory instead.
-     * @expectedDeprecation Storing templates in the "%ssf_translation%s/Resources/views" directory is deprecated since Symfony 4.2, use the "%ssf_translation%s/templates" directory instead.
+     * @expectedDeprecation Loading Twig templates from the "%ssf_translation%s/Resources/views" directory is deprecated since Symfony 4.2, use the "%ssf_translation%s/templates" directory instead.
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationDebugCommandTest.php]: messages were clarified in #30952

File: src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig [hunk 2]
```diff
@@ -114,7 +114,6 @@
     <div class="form-group{% if (not compound or force_error|default(false)) and not valid %} has-error{% endif %}">
         {{- form_label(form) }} {# -#}
         {{ form_widget(form, widget_attr) }} {# -#}
-        {{ form_widget(form) }} {# -#}
```
Comment (xabbuh) [inline src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig]: probably just a mistake when resolving conflicts while merging up

File: src/Symfony/Component/Form/Tests/Extension/Core/Type/FileTypeTest.php [hunk 3]
```diff
@@ -203,7 +203,7 @@ public function testFailedFileUploadIsTurnedIntoFormErrorUsingHttpFoundationRequ
             ->createBuilder(static::TESTED_TYPE)
             ->setRequestHandler(new HttpFoundationRequestHandler())
             ->getForm();
-        $form->submit(new UploadedFile(__DIR__.'/../../../Fixtures/foo', 'foo', null, null, $errorCode, true));
+        $form->submit(new UploadedFile(__DIR__.'/../../../Fixtures/foo', 'foo', null, $errorCode, true));
```
Comment (xabbuh) [inline src/Symfony/Component/Form/Tests/Extension/Core/Type/FileTypeTest.php]: adjustment was needed after merging #30895 up into the `4.2` branch as the filesize argument was deprecated in the meantime

**Review by vudaltsov — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Component/Form/Tests/Extension/Core/Type/FileTypeTest.php [hunk 4]
```diff
@@ -316,15 +318,21 @@ public function uploadFileErrorCodes()
...
     }
 
-    private function createUploadedFile(RequestHandlerInterface $requestHandler, $path, $originalName)
+    private function createUploadedFile(RequestHandlerInterface $requestHandler, $path, $originalName, $errorCode = 0)
     {
         if ($requestHandler instanceof HttpFoundationRequestHandler) {
-            return new UploadedFile($path, $originalName, null, null, true);
+            $class = new \ReflectionClass(UploadedFile::class);
+
+            if ($class->getConstructor()->getNumberOfParameters() === 5) {
```
Comment (derrabus) [inline src/Symfony/Component/Form/Tests/Extension/Core/Type/FileTypeTest.php]: Yoda style

**Review by HeahDude — APPROVED** [review]
Thank you!

**Pull request merged**